test/CMakeLists.txt

```diff
@@ -17,7 +17,11 @@ project(manifold_test)
 set(CMAKE_CXX_STANDARD 17)
 enable_testing()
 
-set(SOURCE_FILES test_main.cpp polygon_test.cpp cross_section_test.cpp properties_test.cpp manifold_test.cpp hull_test.cpp smooth_test.cpp sdf_test.cpp boolean_test.cpp boolean_complex_test.cpp samples_test.cpp)
+set(SOURCE_FILES test_main.cpp polygon_test.cpp sdf_test.cpp boolean_test.cpp)
+
+if(MANIFOLD_CROSS_SECTION)
+  list(APPEND SOURCE_FILES boolean_complex_test.cpp cross_section_test.cpp hull_test.cpp manifold_test.cpp properties_test.cpp samples_test.cpp smooth_test.cpp)
```

Comment:
How many of the tests in these files actually require cross section? I can see skipping `cross_section_test.cpp`, but it feels a bit like throwing the baby out with the bathwater to remove all of these. Can we just `#ifdef` around a few tests in each file?